src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json

```diff
@@ -965,6 +964,110 @@
 						"Action": "execute"
 					}
 				]
+			},
+			{
+				"Sequence": 3,
+				"Name": "placeNode",
+				"Type": "tooltip",
+				"ShowLibrary": true,
+				"TooltipPointerDirection": "none",
+				"Width": 480,
+				"Height": 248,
+				"StepContent": {
+					"Title": "OnboardingPlaceNodeTitle",
+					"FormattedText": "OnboardingPlaceNodeText"
+				},
+				"HostPopupInfo": {
+					"VerticalPopupOffset": 300,
+					"HorizontalPopupOffset": 100,
+					"PopupPlacement": "right",
+					"HostUIElementString": "sidebarGrid",
+					"WindowName": "LibraryView",
+					"HighlightRectArea": {
+						"HighlightColor": "#EF9362",
+						"UIElementTypeString": "WebBrowser",
+						"UIElementGridContainer": "sidebarGrid",
+						"WindowElementNameString": "ByCenterPointRadius",
+						"WindowName": "LibraryView"
+					},
+					"CutOffRectArea": {
+						"WidthBoxDelta": 0,
+						"HeightBoxDelta": 0
+					}
+				},
+				"UIAutomation": [
```

Comment:
Can you remind me what this block does?

Reply:
Hi @QilongTang, this block is about automating actions in this step. For example, disabling the next button, uncollapse a menu item, subscribing to a specific button in the workspace...